### PR TITLE
fix: return type of Accuracy#forwad_cpu() norm to array fix#97

### PR DESCRIPTION
**Pull request by kikusu**
Sorry, I found bug my PR(#97). I was fixed return type of Accuracy#cpu_forwad() like SoftmaxCrossEntropy#foward_cpu().

But  I fear that the different return type forward_cpu() and forward_gpu()
```
x_c = Variable( numpy.array( [ [0. , 1.], [0., 1.,] ], dtype=numpy.float32 ) )
t_c = Variable( numpy.array( [ 1, 1], dtype=numpy.int32 ) )
x_g = Variable( cuda.to_gpu( numpy.array( [ [0. , 1.], [0., 1.,] ], dtype=numpy.float32 ) ) )
t_g = Variable( cuda.to_gpu( numpy.array( [1, 1], dtype=numpy.int32 ) ) ) 

cpu = F.accuracy(x_c, t_c).data
gpu = cuda.to_gpu( F.accuracy(x_g, t_g).data )
print cpu, gpu #=> [ 1.] 1.0
print cpu.shape, gpu.shape #=> (1,) ()

cpu = F.softmax_cross_entropy(x_c, t_c).data
gpu = cuda.to_gpu( F.softmax_cross_entropy(x_g, t_g).data )
print cpu, gpu #=> [ 0.31326166] 0.313261657953
print cpu.shape, gpu.shape #=> (1,) ()
```

### CODE REVIEW

**Comment by unnonouno** [comment]
Thank you very much! I think it's a problem of numpy....

```
>>> a = numpy.array(1, numpy.float32)
>>> a
array(1.0, dtype=float32)
>>> a + a
2.0  # ???
>>> g = chainer.cuda.to_gpu(a)
>>> g
array(1.0, dtype=float32)
>>> g + g
array(2.0, dtype=float32)
```

I like the same return types of `forward_cpu` and `forward_gpu` as you write.
Another solution is that `forward_gpu` also returns an array whose shape is `(1,)`.

**Comment by unnonouno** [comment]
Please try the master branch. I fixed `__add__` operator in #108 for this problem.

**Comment by kikusu** [comment]
This problem was solved. Thank you.
